pyuvdata/uvdata/uvdata.py

```diff
@@ -5315,14 +5315,13 @@ def _apply_w_proj(self, new_w_vals, old_w_vals, select_mask=None):
         new_w_vals: float or ndarray of float
             New w-coordinates for the baselines, in units of meters. Can either be a
             solitary float (helpful for unphasing data, where new_w_vals can be set to
-            0.0) or an array of shape (Nselect,) (which is Nblts if select_mask=None).
+            0.0) or an array of shape (Nblts,).
         old_w_vals: float or ndarray of float
             Old w-coordinates for the baselines, in units of meters. Can either be a
             solitary float (helpful for unphasing data, where new_w_vals can be set to
-            0.0) or an array of shape (Nselect,) (which is Nblts if select_mask=None).
+            0.0) or an array of shape (Nblts,).
         select_mask: ndarray of bool
-            Array is of shape (Nblts,), where the sum of all enties marked True is
-            equal to Nselect (mentioned above).
+            Array is of shape (Nblts,), which identifies which records to change.
```

Comment:
Array is of -> Array of

Reply:
Fixed in 4190b5a0350e04050230984dcc531890e5c7d023